common/src/main/java/io/druid/guice/DruidSecondaryModule.java

```diff
@@ -75,24 +76,36 @@ public void configure(Binder binder)
     binder.bind(JsonConfigurator.class).toInstance(jsonConfigurator);
   }
 
-  @Provides @LazySingleton @Json
-  public ObjectMapper getJsonMapper(final Injector injector)
+  @Provides
+  @LazySingleton
+  public InjectableValues getGuiceInjectableValues(final Injector injector)
```

Comment:
Technically no because the ObjectMapper::copy() was deleted. IMHO it makes the later part of that file cleaner at the expense of having a very confusing. But please limit that discussion to https://github.com/druid-io/druid/pull/1092